cmd/supernode/app/root.go

```diff
@@ -145,12 +145,17 @@ func runSuperNode() error {
 
 // initLog initializes log Level and log format of daemon.
 func initLog() error {
-	logPath := path.Join(options.HomeDir, "logs", "app.log")
-	err := dflog.InitLog(options.Debug, logPath, fmt.Sprintf("%d", os.Getpid()))
-	if err != nil {
-		logrus.Errorf("failed to initialize logs: %v", err)
+	logFilePath := path.Join(options.HomeDir, "logs", "app.log")
+
+	opts := []dflog.Option{
+		dflog.WithLogFile(logFilePath),
+		dflog.WithSign(fmt.Sprintf("%d", os.Getpid())),
+		dflog.WithDebug(options.Debug),
```

Comment:
Do we also need to make supernode output logs to console as an default option?

Reply:
I think so. But currently there's no configuration to disable console log. Should we provide options to turn it off?